test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ListTest.scala

```diff
@@ -436,6 +438,91 @@ trait ListTest extends CollectionTest {
       assertTrue(ll.isEmpty())
     }
   }
+
+  @Test def sortWithNaturalOrdering(): Unit = {
+    testSortWithNaturalOrdering[CustomComparable](new CustomComparable(_), false)
+    testSortWithNaturalOrdering[jl.Integer](jl.Integer.valueOf)
+    testSortWithNaturalOrdering[jl.Long](_.toLong)
+    testSortWithNaturalOrdering[jl.Double](_.toDouble)
+  }
+
+  @Test def sortWithComparator(): Unit = {
+    testSortWithComparator[CustomComparable](new CustomComparable(_),
+        (x, y) => x.compareTo(y), false)
+    testSortWithComparator[jl.Integer](_.toInt, (x, y) => x.compareTo(y))
+    testSortWithComparator[jl.Long](_.toLong, (x, y) => x.compareTo(y))
+    testSortWithComparator[jl.Double](_.toDouble, (x, y) => x.compareTo(y))
+  }
+
+  private def testSortWithNaturalOrdering[T <: AnyRef with Comparable[T]: ClassTag](
+      toElem: Int => T, absoluteOrder: Boolean = true): Unit = {
```

Comment:
The `absoluteOrder` parameter seems to be never set.

Reply:
It was set in two places, but not with the named parameter syntax. I've changed the call sites to use named parameters so that this is clearer.